src/constants/DefaultValues.ts

```diff
@@ -47,3 +50,8 @@ export const AMOUNT = {
 
 export const FILE_MAX_SIZE = 1 * 1024 * 1024;
 export const PREPARING_FIELDS_LIMIT = 50;
+
+export const TIME = {
+  MIN: 0,
+  MAX: 100
```

Comment:
why max is 100?

Reply:
I don't have any concrete reason why I chose 100 and not any other value. I just thought that it could work since this value is assigned to both the minutes and hours fields. While 100 minutes is somewhat acceptable, 100 hours, on the other hand, well... :sweat_smile:
I don't know what other value could work.